README.md

```diff
@@ -5,12 +5,27 @@ ArkhamDB
 
 This guide assumes you know how to use the command-line and that your machine has php and mysql installed.
 
+## Pre-requisites
+
+- PHP (version: ~7)
+- Composer (version: 1)
+- Various extra PHP modules, such as:
+  - php-curl
+  - pdo-mysql
+  - mysqli
+  - mysqlnd
+  - zlib
+- MySQL (or MariaDB)
+
+
+## Install
+
 - install composer: https://getcomposer.org/download/
-- clone the repo somewhere
-- cd to it
+  - NOTE: use version 1.10.26 (the last release of version 1), using `php composer-setup.php --version=1.10.26`
```

Comment:
Thanks a lot for the PR/issue/..., as I've been struggling with the same issues! ;) 
Instead of `--version=1.10.26`, you can use `--1` though I doubt they'll ever make a new Composer 1 release.